pkg/provider/acceptance_connection_confluent_schema_registry_test.go

```diff
@@ -0,0 +1,128 @@
+package provider
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/MaterializeInc/terraform-provider-materialize/pkg/materialize"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/jmoiron/sqlx"
+)
+
+func TestAccConnConfluentSchemaRegistry_basic(t *testing.T) {
+	connectionName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConnConfluentSchemaRegistryResource(connectionName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckConnConfluentSchemaRegistryExists("materialize_connection_confluent_schema_registry.test"),
```

Comment:
There are some other attributes it would be good to run tests for. It can be difficult with some of the nested attributes.